docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -0,0 +1,791 @@
+---
+title: Migrating from MvvmLight
+author: jamesmcroft
+description: This article describes how to migrate MvvmLight solutions to the Windows Community Toolkit MVVM framework.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, mvvm, mvvmlight, net core, net standard
+dev_langs:
+  - csharp
+---
+
+# Migrating from MvvmLight
+
+This article outlines some of the key differences between the [MvvmLight Toolkit](https://github.com/lbugnion/mvvmlight) and the MVVM Toolkit to ease your migration. 
+
+While this article specifically focuses on the migrations from MvvmLight to the MVVM Toolkit, note that there are additional improvements that have been made within the MVVM Toolkit so I highly recommend taking a look at the documentation for the APIs. 
+
+## Installing the WCT MVVM Toolkit
+
+To take advantage of the Windows Community Toolkit MVVM framework, you'll first need to install the latest NuGet package to your existing Windows application.
+
+### Install via .NET CLI
+
+```
+dotnet add package Microsoft.Toolkit.Mvvm --version x.x.x
+```
+
+### Install via PackageReference
+
+```
+<PackageReference Include="Microsoft.Toolkit.Mvvm" Version="x.x.x" />
+```
+
+## Migrating ObservableObject
+
+The following steps focus on migrating your existing components which take advantage of the `ObservableObject` of the MvvmLight Toolkit. 
+
+The Windows Community Toolkit MVVM framework provides an [`ObservableObject`](ObservableObject) type that is similar. 
+
+The first change here will be swapping using directives in your components.
+
+```csharp
+// MvvmLight
+using GalaSoft.MvvmLight;
+
+// Toolkit.Mvvm
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+```
+
+Below are a list of migrations that will need to be performed if being used in your current solution.
+
+### ObservableObject Methods
+
+#### VerifyPropertyName ( string )
+
+There is no direct replacement for the `VerifyPropertyName(string)` method and any code using this should be altered or removed.
```

Comment:
We might want to mention that the point of this method was to support the usage of explicit property names (as in, passed as strings), which were error prone and might have led to eg. failed manual refactorings. I would argue there's really no need for an alternative for this anymore, as we now have the `nameof` operator directly, which completely solves this (since it produces a strong symbolic link to the target properly, so even when doing refactoring all the various occurrences will be updated automatically as well).

I mean, I think we should explain the rationale for its removal here?
And also why users shouldn't worry about it being missing 😊

Reply:
I've dropped in some notes around the use of the `nameof` keyword